src/main/java/entities/login_leaderboard/User.java

```diff
@@ -4,5 +4,43 @@
  * The user stores a user's name, password, email, and the quickest
  * easy, medium, hard scores.
  */
-public class User {
+public class User{
+    private final String username;
```

Comment:
We decided to follow the convention to write final variables in caps. So username should be changed to USERNAME and password to PASSWORD